pages/admin/datasets/new.vue

```diff
@@ -178,7 +178,7 @@ async function updateDataset(asPrivate: boolean) {
     }
   }
 
-  await navigateTo(newDataset.value.page, { external: true })
+  await navigateTo(newDataset.value.page)
```

Comment:
Using `.page` without external will break on local

Reply:
I just tested and the redirects seems to work, even to localhost:7000 and without the external